src/pages/(dashboard)/profile/edit/page.tsx

```diff
@@ -1,18 +1,191 @@
+			</div>
+
 			<Button
 				unstyled={true}
-				className="mx-5 mt-4 flex size-11 items-center justify-center self-start rounded-full
-					bg-grey-100"
-				onClick={() => history.back()}
+				className="mx-auto w-fit rounded-[24px] bg-white px-[10px] py-[8px] text-[14px] font-medium"
 			>
-				<ArrowIcon className="text-icon" />
+				Change Photo
 			</Button>
+
+			<Form.Root methods={methods} className="gap-4">
+				<Form.Item control={control} name="firstname" className="space-y-3">
+					<Form.Label className="font-medium">First name</Form.Label>
+					<Form.Input
+						className="h-[44px] w-full rounded-[8px] border border-grey-200 px-[10px]
+							placeholder:text-grey-600"
+						placeholder="Enter new details"
+					/>
+				</Form.Item>
+
+				<Form.Item control={control} name="lastname" className="space-y-3">
+					<Form.Label className="font-medium">Last name</Form.Label>
+					<Form.Input
+						className="h-[44px] w-full rounded-[8px] border border-grey-200 px-[10px]
+							placeholder:text-grey-600"
+						placeholder="Enter new details"
+					/>
+				</Form.Item>
+
+				<div className="space-y-3">
+					<Form.Item control={control} className="space-y-3" name="phone">
+						<Form.Label className="font-medium">Phone number</Form.Label>
+
+						<div className="flex h-[44px] items-center rounded-[8px] border border-grey-200 px-[10px]">
+							<p
+								className="relative flex cursor-pointer items-center gap-[2px] pr-[12px]
+									text-[14px] font-normal"
+								onClick={toggleOptions("phone")}
+							>
+								Mobile
+								<IconBox icon="solar:alt-arrow-down-outline" className="text-grey-700" />
+								{isShow === "phone" && (
+									<div
+										className={`absolute left-[-10px] top-full w-[100px] bg-white px-3 py-2
+										outline-none`}
+									>
+										<p className="py-2">Mobile</p>
+										<p className="py-2">Home</p>
+										<p className="py-2">Work</p>
+									</div>
+								)}
+							</p>
+							<Form.Input
+								type="tel"
+								placeholder="+234 7062462466"
+								className="flex-1 bg-transparent outline-none"
+							/>
+						</div>
+					</Form.Item>
+
+					<div>
+						<Button
+							unstyled={true}
+							className="flex items-center gap-[2.83px] font-medium text-primary"
+						>
+							<IconBox
+								icon="solar:add-circle-outline"
+								className="size-[13.33px] font-medium text-primary"
+							/>
+							Add Field
+						</Button>
+					</div>
+				</div>
+
+				<div className="space-y-3">
+					<Form.Item control={control} className="space-y-3" name="email">
+						<Form.Label className="font-medium">Email</Form.Label>
+
+						<div className="flex h-[44px] items-center rounded-[8px] border border-grey-200 px-[10px]">
+							<p
+								className="relative flex cursor-pointer items-center gap-[2px] pr-[12px]
+									text-[14px] font-normal"
+								onClick={toggleOptions("email")}
```

Comment:
Switch these custom toggles with the dropdown components that are already available